packages/ast-spec/src/type/TSImportType/fixtures/type-import-type-with-import-attributes/fixture.ts

```diff
@@ -1,3 +1,4 @@
 // TODO: This fixture might be too large, and if so should be split up.
 
 type A = import("A", { with: { type: "json" } });
+type B = import("B", { assert: { type: "json" } });
```

Comment:
> the test in the right place.

You're close 🙂. These ast-spec fixtures are the right place in general, but we generally try to keep the fixtures small and single-purpose. I think the right step here would be to:

* Copy this `type-import-type-with-import-attributes` folder to a new one
* Have one of these fixtures test `assert` and the other test `with`
* Remove the `// TODO: ` comments from each - those were automatically added and I think they're not accurate in this case

Reply:
Addressed in 725831a58638ec62d0316db26d2a80e7e883639c.